test/multiapi/multiapiTests.ts

```diff
@@ -0,0 +1,15 @@
+import * as assert from "assert";
+
+import * as vDefault from "./generated";
```

Comment:
I'm concerned that these tests will require a multiapi build before they pass. The dependencies in gulp are just gonna start looking a bit wacky.